src/SelfMeasurements-2-0/Cert/_reference/resources-serve-receive/medmij-selfmeasurements-serve-receive-scenario-2-3-bundle.xml

```diff
@@ -1475,4 +1475,60 @@
             <url value="Observation"/>
         </request>
     </entry>
+    <!--medmij-selfmeasurements-HeartRate-kwalificatie5-->
+    <entry>
+        <fullUrl value="urn:oid:2.16.840.1.113883.2.4.3.11.9999.14"/>
+        <resource>
+            <Observation>
+                <meta>
+                    <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-HeartRate"/>
+                </meta>
+                <status value="final"/>
+                <category>
+                    <coding>
+                        <system value="http://hl7.org/fhir/observation-category"/>
+                        <code value="vital-signs"/>
+                        <display value="Vital Signs"/>
+                    </coding>
+                </category>
+                <code>
+                    <coding>
+                        <system value="http://loinc.org"/>
+                        <code value="8867-4"/>
+                        <display value="Heart rate"/>
+                    </coding>
+                </code>
+                <subject>
+                    <reference value="urn:oid:2.16.840.1.113883.2.4.3.11.9999.1"/>
+                    <display value="Marieke XXX_Bergzoon"/>
+                </subject>
+                <effectiveDateTime value="${DATE, T, D, -22}T08:20:00+02:00"/>
+                <performer>
+                    <reference value="urn:oid:2.16.840.1.113883.2.4.3.11.9999.1"/>
+                    <display value="Marieke XXX_Bergzoon"/>
+                </performer>
+                <valueQuantity>
+                    <value value="93"/>
+                    <unit value="/min"/>
+                    <system value="http://unitsofmeasure.org"/>
+                    <code value="/min"/>
+                </valueQuantity>
+                <interpretation>
+                    <coding>
+                        <system value="http://snomed.info/sct"/>
+                        <code value="271636001"/>
+                        <display value="Hartslag regelmatig"/>
+                    </coding>
+                </interpretation>
+                <comment value ="Na Traplopen"/>
+                <method>
+                    <coding>
+                        <system value="http://snomed.info/sct"/>
+                        <code value="113011001"/>
+                        <display value="Palpatie"/>
+                    </coding>
+                </method>
+            </Observation>
+        </resource>
+      </entry>    
```

Comment:
Dit lijkt mij niet te werken, is dit getest? Er moet nog een request.method bij.

Reply:
In Touchstone getest, maar geen foutmelding gekregen. Ik heb de request.method toegevoegd.

Reply:
@eduardderijcke , dit ging niet goed, kijk maar eens naar bij de response voor heart reate. https://touchstone.aegis.net/touchstone/scriptexecution?exec=202103120947498481426502&qn=/FHIRSandbox/Nictiz/dev/FHIR3-0-2-MM202001-Cert/SelfMeasurements-2-0/XIS-Server/selfmeasurements-xis-2-3-send-observations-from-period-xml#WAITING_OPER

Kan je het nu nog eens testen?

Reply:
Miss heb je dat al gedaan, maar iig nog een Touchstone testexecution doen.

Reply:
Gedaan en goed resultaat gekregen:

 <response>
      <status value="201"/>

Reply:
Top!